.ci/scripts/install-tools.bat

```diff
@@ -15,8 +15,8 @@ REM Set the USERPROFILE to the previous location to fix issues with chocolatey i
 SET PREVIOUS_USERPROFILE=%USERPROFILE%
 SET USERPROFILE=%OLD_USERPROFILE%
 IF NOT EXIST C:\Python38\python.exe (
```

Comment:
Could you check - if `C:\Python38\python.exe` to be changed to `C:\Python311\python.exe` or similar?